taskcluster/fenix_taskgraph/transforms/signing.py

```diff
@@ -58,6 +58,8 @@ def set_signing_type(config, tasks):
                 signing_type = "fennec-production-signing"
             elif task["attributes"]["build-type"] in ("nightly", "android-test-nightly"):
                 signing_type = "production-signing"
+            elif task["attributes"]["build-type"] in ("beta-mozillaonline", "release-mozillaonline"):
```

Comment:
It looks like if you follow my advice on the other PR, we'll adjust the config or logic for `set_signing_format`.

Reply:
(If this is difficult, we can certainly use a new cert type :)

Reply:
Looks like it was easy to do `by-build-type` in the kind.